opencensus/trace/exceptions_status.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.rpc import code_pb2
+from opencensus.trace.status import Status
+
+DNS_RESOLUTION_FAILED = Status(code_pb2.UNKNOWN)
```

Comment:
Where is `DNS_RESOLUTION_FAILED` used? Setting it to gRPC `UNKNOWN` could be misleading.

Reply:
Is isn’t used but is listed in the [specs](https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/HTTP.md)!

Reply:
I see, it looks like this section in the specs is giving example rather than a comprehensive list of what should be handled/implemented.
I don't have strong opinion here, feel free to decide whether to keep it or not.

Reply:
Yeah, they are just examples and I don’t agree with the unknown value also, but I think with OpenTelemetry we must have a stronger opinion about this! Will remove it and push.